src/contents/all/components/knot/KnotInput/KnotInput.scss

```
@@ -8,14 +8,15 @@
 }
 
 .DraftEditor-editorContainer {
+  border: 0px;
```

replace with:
```suggestion
  border: 0;
```

(no need for `px`)